packages/installer/deploy.sh

```diff
@@ -89,6 +93,8 @@ mount_image() {
         mount_state
     fi
 
+    is_mounted /usr/local || mount ${PERSISTENT} /usr/local
+
```

Comment:
should we umount this on `clean2` just in case?

Reply:
good question :thinking: It should not do any harm... on the other hand it might be helpful for the hooks if they have to do something in the COS_PERSISTENT partition